soh/soh/Enhancements/audio/AudioCollection.cpp

```diff
@@ -8,301 +8,323 @@
 #include <locale>
 #include <filesystem>
 
-#define SEQUENCE_MAP_ENTRY(sequenceId, label, sfxKey, category) \
-    { sequenceId, { sequenceId, label, sfxKey, category } }
+#define SEQUENCE_MAP_ENTRY(sequenceId, label, sfxKey, category, canBeReplaced, excludeAsReplacement) \
```

Comment:
I think it's a little weird that one bool says that something *can* be replaced and another says that it *can't* be used as a replacement, I think this could lead to errors like the warp song text's "mysterious" option which people thought was the opposite of what it actually was

Reply:
I couldn't really think of good names for them to have them both be mean "off" and "on", so I'm open for names if you got them.

Reply:
`canBeReplaced` & `canReplace` seem like a simple solution to me, although they are similar and it may get confusing

Reply:
I could see `canBeReplaced` and `canReplace` being interpreted to mean the same thing. Even if it's a bit verbose, `canBeReplaced` and `canBeUsedAsReplacement` should work.

Reply:
@aMannus do we want to make the changes here ^ or merge as is?

Reply:
I should make the changes first, didn't have time then forgot. Afraid it'll have to wait till next week though most likely.

Reply:
Updated.